app/views/further_education_payments/provider/claims/sign_in.html.erb

```diff
@@ -0,0 +1,37 @@
+<% content_for(
+  :page_title,
+  page_title(
+    "Verify a further education retention payment",
+    journey: current_journey_routing_name
+  )
+) %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      Verify a further education retention payment
+    </h1>
+
+    <p class="govuk-body govuk-!-margin-top-6">
+      Use this service to verify a retention payment for further education
+      teachers.
+    </p>
+
+    <p class="govuk-body govuk-!-margin-top-6">
+      You will need to log in using DfE Sign-in. If you don't have a DfE Sign-in
+      account yet, we will help you create one.
+    </p>
+
+    <%= button_to(
+      "/further-education-payments-provider/auth/dfe_fe_provider",
+      class: "govuk-button govuk-button--start",
+      data: {
+        module: "govuk-button"
+      }
+    ) do %>
+      Start now
+      <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" aria-hidden="true" focusable="false">
+        <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
+      </svg>
+    <% end %>
```

Comment:
don't bother changing but fyi for next time https://govuk-components.netlify.app/components/start-button/